src/main/resources/assets/thaumcraft/research/new.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
this file should not be here

Reply:
removed in commit: [https://github.com/CleanroomMC/GroovyScript/commit/966ea430c0791ce980fb21ed8ca62d723870071a](https://github.com/CleanroomMC/GroovyScript/commit/966ea430c0791ce980fb21ed8ca62d723870071a)